pyronear/datasets/wildfire/wildfire.py

```diff
@@ -28,7 +29,7 @@ class WildFireDataset(Dataset, VisionMixin):
         If left to None, will be set to ['fire']
         Example: ['fire', 'clf_confidence', 'loc_confidence', 'x', 'y']
 
-    path_to_frames: str
+    path_to_frames: path type
```

Comment:
It could be great to ensure retro-compatibility and flexibility: being able to accept `strings` and `Path-like` objects!

Reply:
Thank you for the comment but I don't see a use case where strings would be useful ?

Reply:
For instance, if you are loading project parameters from config file (such as path to frames), you would get a string. In this case, it would be handy to not have to convert it to a Path-like object but directly pass it to the WildFire class initialisation

Reply:
Understood, adding changes in next commit